database/database.go

```diff
@@ -0,0 +1,45 @@
+package database
+
+import (
+	sqlite "gorm.io/driver/sqlite"
+	gormpkg "gorm.io/gorm"
+)
+
+const (
+	POSTGRES = "postgres"
+	SQLITE   = "sqlite"
+)
+
+type Options struct {
+	Filename string `json:"filename,omitempty"`
+	Engine   string `json:"engine,omitempty"`
+}
+
+type Model struct {
+	ID        string `json:"id,omitempty" gorm:"primarykey"`
+	CreatedAt string `json:"created_at,omitempty" gorm:"index"`
+	UpdatedAt string `json:"updated_at,omitempty" gorm:"index"`
+	DeletedAt string `json:"deleted_at,omitempty" gorm:"index"`
```

Comment:
Does this mean that deleted records are not removed from a table, i.e. actually not deleted?